Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -489,6 +489,17 @@ extension FirefoxHomeViewController: UICollectionViewDelegateFlowLayout {
 
     fileprivate func showSiteWithURLHandler(_ url: URL) {
         let visitType = VisitType.bookmark
+        
+        switch url.absoluteString {
+        case Environment.current.blog.absoluteString,
+             Environment.current.financialReports.absoluteString,
+             Environment.current.privacy.absoluteString,
+             Environment.current.howEcosiaWorks.absoluteString:
+            Analytics.shared.open(topSite: url.absoluteString)
```

Comment:
I wonder if we could find a more elegant solution to track one of  the titles instead of the url. Otherwise we'll have many different variations of the urls in all the languages as properties. This will be quite tedious to evaluate.

Reply:
Aren't titles being localised? The language in the URL is something we don't want to track? Example: is the blog more popular in french than in english? These are just speculations but then again we may need to define what we want or not.

Reply:
I mean the ticket states "metadata" that should be tracked. I would also be fine by assigning an enum of strings to all the four different urls .e.g "privacy, info, finance, blog" or similar. Let's check with PM 👍

Reply:
Sure the ticket says title, but again, the title is being localised. An enum could work, but if the URL works for our purposes then no need to add duplication. I'm still waiting for an answer from the PM.